deps/scripts/init_accounts.js

```diff
@@ -0,0 +1,203 @@
+    return balance.balance;
+  }
+
+  async bridgeToChain(l1_bridge_address, starknet_account_address, eth_token_address) {
+    console.log(`🌉 Bridging funds to ${starknet_account_address}...`);
+    const contract = new ethers.Contract(
+      l1_bridge_address,
+      ["function deposit(uint256, uint256)"],
+      this.wallet
+    );
+
+    const initial_balance = await this.getAppChainBalance(
+      starknet_account_address,
+      eth_token_address
+    );
+
+    console.log("Initial balance: ", initial_balance)
+    const amount = "10";
+    const amount_with_fees = (parseFloat(amount) + 0.01).toString();
+    const tx = await contract.deposit(
+      ethers.parseEther(amount),
+      starknet_account_address,
+      { value: ethers.parseEther(amount_with_fees) }
+    );
+    await tx.wait();
+    console.log("✅ Successfully sent ${} ETH on L1 bridge", amount);
+
+    // Wait for funds to arrive on Starknet
+    let counter = 10;
+    while (counter--) {
+      const final_balance = await this.getAppChainBalance(
+        starknet_account_address,
+        eth_token_address
+      );
+      if (final_balance > initial_balance) {
+        console.log(
+          "💰 Account balance:",
+          (final_balance / 10n ** 18n).toString(),
+          "ETH"
+        );
+        return true;
+      }
+      console.log("🔄 Waiting for funds to arrive on Starknet...");
+      await new Promise((resolve) => setTimeout(resolve, 5000));
+    }
+    throw new Error("Failed to bridge funds to Starknet");
+  }
+
+  generateAccountKeys() {
+    const privateKey = starknet.stark.randomAddress();
+    const publicKey = starknet.ec.starkCurve.getStarkKey(privateKey);
+
+    // Calculate the account address
+    const accountConstructorCallData = starknet.CallData.compile({
+      publicKey: publicKey,
+    });
+    const accountAddress = starknet.hash.calculateContractAddressFromHash(
+      publicKey,
+      CONFIG.oz_account_cairo_1_class_hash,
+      accountConstructorCallData,
+      0
+    );
+
+    return {
+      address: accountAddress,
+      privateKey: privateKey,
+      publicKey: publicKey,
+    };
+  }
+
+  async deployAccount(accountKeys) {
+    console.log(`⚙️ Deploying account at ${accountKeys.address}...`);
+    const account = new starknet.Account(
+      this.starknet_provider,
+      accountKeys.address,
+      accountKeys.privateKey,
+      "1"
+    );
+
+    const { transaction_hash } = await account.deployAccount({
+      classHash: CONFIG.oz_account_cairo_1_class_hash,
+      constructorCalldata: [accountKeys.publicKey],
+      addressSalt: accountKeys.publicKey,
+    });
+
+    // Wait for deployment
+    const receipt = await this.starknet_provider.waitForTransaction(transaction_hash);
+    if (!receipt.isSuccess()) {
+      throw new Error(`Failed to deploy account - ${transaction_hash}`);
+    }
+    console.log(`✅ Account deployed successfully - ${transaction_hash}`);
+    return receipt;
+  }
+}
+
+async function main() {
```

Comment:
nice script! 😃 
looks like it can be used for CI tests, maybe dividing the script into two (class def and main program with configs) could be easier to use in CI envs. What do you think?